libraries/math/src/processor.rs

```diff
@@ -104,6 +114,22 @@ pub fn process_instruction(
             msg!("{}", result as u64);
             Ok(())
         }
+        MathInstruction::F32Exponentiate { base, exponent} => {
+            msg!("Calculating f32 Exponentiate");
```

Comment:
nit: maybe make this clearer?
```suggestion
            msg!("Calculating f32 exponent");
```